pkgs/by-name/aw/aws-gate/package.nix

```diff
@@ -23,6 +23,7 @@ python3Packages.buildPythonApplication rec {
 
   postPatch = ''
     rm aws_gate/bootstrap.py tests/unit/test_bootstrap.py
+    echo > requirements/requirements.txt
```

Comment:
consider using `pythonRelaxDepsHook`

https://nixos.org/manual/nixpkgs/stable/#using-pythonrelaxdepshook

Reply:
Makes sense that there would already be a tool for this 😅 
Thank you for the suggestion, added.